sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1278,6 +1268,13 @@ def post_port_active_apsel_to_db(self, api, lport, host_lanes_mask):
 
         asic_index = self.port_mapping.get_asic_id_for_logical_port(lport)
         intf_tbl = self.xcvr_table_helper.get_intf_tbl(asic_index)
+        if not intf_tbl:
+            self.log_error("Active ApSel db update: intf_tbl not found for {}".format(lport))
+            return
+        found, _ = intf_tbl.get(lport)
+        if not found:
+            helper_logger.log_debug("Active ApSel db update: {} not found in INTF_TABLE".format(lport))
```

Comment:
@mihirpat1 can we do warn so that its there in log?

Reply:
@prgeor I have replaced this with warning now